src/status_im/ui/market/subs.cljs

```diff
@@ -0,0 +1,38 @@
+(ns status-im.ui.market.subs
+  (:require
+    [clojure.string :as string]
+    [re-frame.core :as rf]
+    [status-im.contexts.wallet.common.utils :as wallet.utils]
+    [utils.number]
+    [utils.string]))
+
+(rf/reg-sub
+ :ui.market/leaderboard
+ :<- [:domain.market/leaderboard]
+ (fn [leaderboard]
+   (->> leaderboard
+        (sort-by val)
```

Comment:
There's a nice data structure and lib for CLJS called a priority map https://github.com/tailrecursion/cljs-priority-map, which supports efficient sorting by value (not by key as in clj core `sorted-map`). I can't recall the places I've seen in the repo that could use this, but I'm pretty sure it's not the first time I see an opportunity for using this data structure. Performance could be quite good (could be even better than manual sorting all the time), but should be benchmarked.

cc @ulisesmac @flexsurfer